### PR TITLE
Landing Page Prototype

### PR DESCRIPTION
**Pull request by Kelvin229**
## Overview
This is the prototype of the Figma design Gideon created earlier. We can discuss it in the next meeting.

## Changes
- Added a responsive Navbar with authentication links.
- Created the testimonials section.

Lmk if you have any questions!

### CODE REVIEW

File: src/components/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+type ButtonProps = {
+  href: string;
+  title: string;
+};
+
+const Button = ({ href, title }: ButtonProps) => {
+  return (
+    <Link href={href}>
+    <a href={href} className="inline-block bg-black text-white py-2 px-6 rounded hover:bg-gray-700 transition duration-300">{title}</a>
+    </Link>
```
Comment (gideonoludeyi) [inline src/components/Button.tsx:12]: I'm getting the following error when running the application. Apparently, Next.js doesn't like nesting an `<a>` tag within their `<Link>` component.
```
Error: Invalid <Link> with <a> child. Please remove <a> or use <Link legacyBehavior>.
```

This 👇  fixed the issue on my end
```suggestion
    <Link href={href} className="inline-block bg-black text-white py-2 px-6 rounded hover:bg-gray-700 transition duration-300">{title}</Link>
```

File: src/pages/index.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import Button from '../components/Button';
+import Navbar from '../components/Navbar';
+import Testimonial from '../components/Testimonial';
+import Link from 'next/link';
+
+export default function HomePage() {
+    // We may want to get these testimonials from a static file or API
+  const testimonials = [
+    { quote: 'This app has been a game-changer for my studies.', author: 'Jason Parsons', title: '4th Year CS Student' },
+    { quote: 'This app has been a game-changer for my studies.', author: 'Jenny Ma', title: '4th Year CS Student' },
```
Comment (gideonoludeyi) [inline src/pages/index.tsx:12]: Forgot a group member 🙂
```suggestion
    { quote: 'This app has been a game-changer for my studies.', author: 'Akshay Rash', title: '4th Year CS Student' },
```

File: src/pages/index.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    { quote: 'This app has been a game-changer for my studies.', author: 'Nikos Dritsakos', title: '4th Year CS Student' },
+    { quote: 'This app has been a game-changer for my studies.', author: 'Luke Reinaerts', title: '4th Year CS Student' },
+    { quote: 'This app has been a game-changer for my studies.', author: 'Gideon O', title: '4th Year CS Student' },
```
Comment (gideonoludeyi) [inline src/pages/index.tsx:15]:
```suggestion
    { quote: 'This app has been a game-changer for my studies.', author: 'Gideon Oludeyi', title: '4th Year CS Student' },
```

**Review by gideonoludeyi — COMMENTED** [review]
Can we get a screenshot of the landing page UI?

**Comment by Kelvin229** [comment]
This is a screenshot of the new UI:

image

I also changed the `favicon.ico` for the apps logo

**Review by gideonoludeyi — APPROVED** [review]

**Pull request merged**